csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -149,6 +149,19 @@ public string Format(IMessage message)
         }
 
         /// <summary>
+        /// Formats the specified field of the message as JSON.
```

Comment:
I have commited it as two commits to discuss which one would be better. Right now I would go with the later commit (field only). If you agree with this variant, I would drop the first commit.